src/qinfer/utils.py

```diff
@@ -439,7 +439,53 @@ def compactspace(scale, n):
     logit = logistic(scale=scale).ppf
     compact_xs = np.linspace(0, 1, n + 2)[1:-1]
     return logit(compact_xs)
-           
+
+def to_simplex(y):
+    r"""
+    Interprets the last dimension of ``y`` as stick breaking lengths 
+    in inverse-logit space and returns a non-negative array of 
+    the same shape where the last dimension always sums to unity;
+    simplices.
+
+    Inverse to ``from_simplex``.
```

Comment:
Would it be possible to list a quick example here? I think it might help some users who are less familiar with the notion of using logit space.